Api/ATM/ATM1/Response.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace baibaratsky\WebMoney\Api\ATM\ATM1;
+
+use baibaratsky\WebMoney\Request\AbstractResponse;
+
+/**
+ * Class Response
+ *
+ * @link http://wiki.wmtransfer.com/projects/webmoney/wiki/Interface_ATM1
+ */
+class Response extends AbstractResponse
+{
+    /** @var string payment/@currency */
+    protected $currency;
+
+    /** @var string payment/@exchange */
+    protected $exchange;
+
+    /** @var string payment/purse */
+    protected $payeePurse;
+
+    /** @var float payment/upexchange */
+    protected $upExchange;
+
+    /** @var float payment/course */
```

Comment:
This is mentioned only in russian docs: https://wiki.webmoney.ru/projects/webmoney/wiki/%D0%98%D0%BD%D1%82%D0%B5%D1%80%D1%84%D0%B5%D0%B9%D1%81_ATM1